lib/code_corps/helpers/policy.ex

```diff
@@ -0,0 +1,67 @@
+defmodule CodeCorps.Helpers.Policy do
+  @moduledoc """
+  Holds helpers for extracting record relationships and determining roles for
+  authorization policies.
+  """
+
+  import Ecto.Query
+
+  alias CodeCorps.Organization
```

Comment:
In Elixir 1.3, I believe you can do `alias CodeCorps.{Organization,OrganizationMembership,Project,User,Repo}`